### ISSUE TITLE
Model code different from lua

### ISSUE THREAD

**Issue by elbaro** [1adrianb/face-alignment#69]
Hi, I found a few differences from the Lua model.

- ConvBlock's batchnorm has no bias (affine=False) in Lua
- FullyConnected(1x1 conv) has batchnorm(affine=False) in Lua
- The first pooling is max_pool in Lua while pytorch model uses avg_pool
